packages/models-library/src/models_library/rpc_schemas_clusters_keeper/clusters.py

```diff
@@ -23,3 +24,10 @@ class OnDemandCluster(BaseModel):
     wallet_id: WalletID | None
     dask_scheduler_ready: bool
     eta: datetime.timedelta
+
+
+@dataclass(frozen=True)
+class EC2InstanceType:
```

Comment:
Q: so the idea is to have also separate models for rpc? Up to now I was reusing the ones we have for the rest. In any case we call them "api_schemas".

Q2: why dataclasses and not pydantic? Here using PositiveInt, BtyeSize is merely informative (i.e. for mypy) but you lose the parsing&validation capabiltities

Reply:
yes. I will rename it. Actually it was already there and named this way. I put it like that to differentiate from all the others that are Rest APIs. in any case ok.

I do not use pydantic because they do not need parsing.